wiki/troubleshooting.md

```diff
@@ -17,6 +17,14 @@ To run SFC, from am elevated command prompt run `sfc /scannow`. It is recommend
 
 For more information about SFC, check [this page](https://support.microsoft.com/en-us/help/929833/use-the-system-file-checker-tool-to-repair-missing-or-corrupted-system)
 
+## CHKDSK
+
+This utility checks the file system and file system metadata of a volume for errors.
+To use CHKDSK, open CMD (aka. Command Prompt) with administrator privileges and type `<volume> /f`.
+**Replace `<volume>` with the actual volume you want to scan, such as `C:`.** Press enter to run the command.
+If you are checking drive C:, you may get an warning that CHKDSK cannot lock the partition and the computer needs to restart to continue. To confirm, type `Y` and press enter. CHKDSK will then check the drive the next time you restart your computer.
+For more information about CHKDISK, check [this page](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/chkdsk?tabs=event-viewer)
```

Comment:
```suggestion
For more information about CHKDISK, visit [Microsoft's Documentations](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/chkdsk).
```

Reply:
I am going to be pedantic and say the link text should be `Microsoft's documentation`, not `Microsoft's Documentations`